azext_iot/tests/test_iot_central_int.py

```diff
@@ -400,6 +414,28 @@ def _create_device(self, **kwargs) -> (str, str):
         self.cmd(command, checks=checks)
         return (device_id, device_name)
 
+    def _create_user(self,):
+        user_id = self.create_random_name(prefix="aztest", length=24)
+        email = user_id + "@test.com"
```

Comment:
nit - lets restrict the email domain to `@microsoft` in the off chance that some user really exists (even though its highly unlikely) with `az...@test.com`